src/main/java/net/dv8tion/jda/api/entities/MessageEmbed.java

```diff
@@ -538,6 +540,20 @@ public String getProxyUrl()
             return proxyUrl;
         }
 
+        /**
+         * Returns an {@link AttachmentProxy} for this embed thumbnail
+         *
+         * @return The {@link AttachmentProxy} of this embed thumbnail
+         *
+         * @see    #getProxyUrl()
+         */
+        @Nullable
+        public AttachmentProxy getProxy()
```

Comment:
Are you sure embed URLs use the attachment urls?

Reply:
The proxied URL does use attachment urls, example: https://images-ext-2.discordapp.net/external/nme4UBW_Xauoe9mcu17Mt3rJyIMWrQezqlc8V0LHY24/https/media.giphy.com/media/5i7umUqAOYYEw/giphy.gif?width=64&height=64